github-dark.css

```diff
@@ -1989,7 +1989,7 @@
   }
   .text-pending, a.text-pending, .compare-pr-placeholder p,
   .compare-pr-placeholder .help-link, .text-pending strong {
-    color: #cc5 !important;
+    color: #ddd !important;
   }
```

Comment:
it looks very odd to have yellow letters in this box, how does that look in travis status? can you grab me a screenshot?